Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

```diff
@@ -0,0 +1,441 @@
+    <value>Eléments</value>
+  </data>
+  <data name="MappedPath.Text" xml:space="preserve">
+    <value>Chemin de représentation</value>
+  </data>
+  <data name="Edit.Text" xml:space="preserve">
+    <value>Modifier</value>
+  </data>
+  <data name="RolePermissions.Text" xml:space="preserve">
+    <value>Permissions Rôle</value>
+  </data>
+  <data name="UserPermissions.Text" xml:space="preserve">
+    <value>Permissions utilisateur</value>
+  </data>
+  <data name="FolderId.Text" xml:space="preserve">
+    <value>ID de dossier1</value>
+  </data>
+  <data name="General.Text" xml:space="preserve">
+    <value>Général</value>
+  </data>
+  <data name="Permissions.Text" xml:space="preserve">
+    <value>Permissions</value>
+  </data>
+  <data name="FileId.Text" xml:space="preserve">
+    <value>ID du fichier</value>
+  </data>
+  <data name="ConfirmDeleteMessage.Text" xml:space="preserve">
+    <value>Voulez vous vraiment effacer les éléments sélectionnés ?</value>
+  </data>
+  <data name="ConfirmMoveMessage.Text" xml:space="preserve">
+    <value>Voulez vous vraiment déplacer les éléments sélectionnés ?</value>
+  </data>
+  <data name="ConfirmUnlinkMessage.Text" xml:space="preserve">
+    <value>Voulez vous vraiment délier les dossiers suivantes ?</value>
+  </data>
+  <data name="Delete.Text" xml:space="preserve">
+    <value>Effacer</value>
+  </data>
+  <data name="DeleteItems.Text" xml:space="preserve">
+    <value>Effacer éléments</value>
+  </data>
+  <data name="DeletingItems.Text" xml:space="preserve">
+    <value>Effacement éléments</value>
+  </data>
+  <data name="Files.Text" xml:space="preserve">
+    <value>Fichiers</value>
+  </data>
+  <data name="Folders.Text" xml:space="preserve">
+    <value>Dossiers</value>
+  </data>
+  <data name="Move.Text" xml:space="preserve">
+    <value>Déplacer</value>
+  </data>
+  <data name="MovingItems.Text" xml:space="preserve">
+    <value>Déplacement éléments</value>
+  </data>
+  <data name="No.Text" xml:space="preserve">
+    <value>Non</value>
+  </data>
+  <data name="SelectDestinationFolder.Text" xml:space="preserve">
+    <value>Sélectionnez le dossier destination.</value>
+  </data>
+  <data name="Unlink.Text" xml:space="preserve">
+    <value>Délier</value>
+  </data>
+  <data name="UnlinkFolders.Text" xml:space="preserve">
+    <value>Délier dossier(s)</value>
+  </data>
+  <data name="UnlinkingFolders.Text" xml:space="preserve">
+    <value>Déliage dossier(s)</value>
+  </data>
+  <data name="Yes.Text" xml:space="preserve">
+    <value>Oui</value>
+  </data>
+  <data name="CopyUrl.Text" xml:space="preserve">
+    <value>Copier l'URL</value>
+  </data>
+  <data name="StatusBarMessage.Text" xml:space="preserve">
+    <value>Vue {0} sur {1} élément(s)</value>
+  </data>
+  <data name="Download.Text" xml:space="preserve">
+    <value>Télécharger</value>
+  </data>
+  <data name="Upload.Text" xml:space="preserve">
+    <value>Charger</value>
```

Comment:
```suggestion
    <value>Téléverser</value>
```

Reply:
normalement on utilise le terme téléverser en Français, sauf que dans les autres traductions dnn c'est charger qui est utilisé, es ce qu'il ne vaut pas mieux utiliser la même traduction partout ?

Reply:
Hmm, je préfères vraiment le mot téléverser car is est plus précis, mais en effet le reste dit charger...

Je suis correct avec l'un ou l'autre mais je préfère téléverser même si on doit éventuellement ajuster le reste.